yaksh/models.py

```diff
@@ -70,7 +87,8 @@ class Course(models.Model):
     rejected = models.ManyToManyField(User, related_name='rejected')
     created_on = models.DateTimeField(auto_now_add=True)
     teachers = models.ManyToManyField(User, related_name='teachers')
-
+    is_trial = models.BooleanField(default=False)
```

Comment:
Nice, default value given. 
@all 
Always give default value to the new attribute added, this makes database migration easy.